jest.config.ts

```diff
@@ -16,6 +16,7 @@ module.exports = {
   testResultsProcessor: "jest-junit",
   transform: { "^.+\\.(ts|tsx)$": "ts-jest" },
   testMatch: ["**/*.test.(ts|js)"],
+  setupFilesAfterEnv: ["<rootDir>/src/test/chainSetup.ts"],
```

Comment:
This is cool and super clean, but it's also magic-y. It's not going to be obvious where a test is getting its RPC_URL or what is happening when a there's a problem with snapshots. Is there a way to include chainSetup explicitly in the test files?  If not, some explanation in the readme and in the chainSetup.ts file would help.

Reply:
I agree. I'll try a way to make it an explicit include, but there might be issues with order. Either way I'll make sure to document in the test about both chainSetup and configSetup.